src/classes/User.js

```diff
@@ -0,0 +1,41 @@
+class User {
+  constructor(userData = {}) {
+    this.id = userData.id;
+    this.name = userData.name;
+    this.pantry = userData.pantry;
+    this.recipesToCook = [];
+  }
+
+  addRecipesToCook(recipe) {
+    this.recipesToCook.push(recipe);
```

Comment:
What happens if this method is called and a copy of that recipe is already in the recipesToCook array?

Reply:
wrapped a conditional around the includes statement